src/contracts/Persistence/Content/ContentInfo.php

```diff
@@ -134,6 +136,11 @@ class ContentInfo extends ValueObject
      * @var bool
      */
     public $isHidden = false;
+
+    public function getContentIdentifier(): string
```

Comment:
1. Naming: 
```suggestion
    public function getContentTypeIdentifier(): string
```

2. Please add new, very simple, PHPUnit coverage: `\Ibexa\Tests\Core\Persistence\ValueObject\ContentInfoTest` class
with test, sth like:
```php
$contentTypeIdentifier = 'foo';
$contentInfo = new ContentInfo(['contentTypeIdentifier' => $contentTypeIdentifier]);
self::assertSame($contentTypeIdentifier, $contentInfo->getContentTypeIdentifier());
```